easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PerlModule'
+
+name = 'Bio-DB-HTS'
+version = '3.01'
+
+homepage = 'https://metacpan.org/release/Bio-DB-HTS'
+description = "Read files using HTSlib including BAM/CRAM, Tabix and BCF database files"
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://cpan.metacpan.org/authors/id/A/AV/AVULLO/']
+sources = ['Bio-DB-HTS-%(version)s.tar.gz']
+checksums = ['12a6bc1f579513cac8b9167cce4e363655cc8eba26b7d9fe1170dfe95e044f42']
+
+builddependencies = [('pkgconf', '1.8.0')]
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('BioPerl', '1.7.8'),
+    ('HTSlib', '1.14'),
+]
+
```

Comment:
@boegel please restore the fix added to the older version in https://github.com/easybuilders/easybuild-easyconfigs/pull/12253

Reply:
Nice catch, that's also missing in `Bio-DB-HTS-3.01-GCC-10.2.0.eb` (which is what I started from), so I'll add it there too

Reply:
fixed in 41a7e5dae4

Reply:
Thanks.